scripts/startup.py

```diff
@@ -157,18 +157,25 @@ def parse_env(env=os.environ):
         m = link_pattern.match(var)
         if m:
             service_name = m.group('service_name')
+            service_port = int(m.group('service_port'))
```

Comment:
Any reason in particular you're casting to an `int`, versus checking if `service_port != '80'` later?